### ISSUE TITLE
[Feature]: Add XLSX report to Karma

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#1484]
### Project

karma accessibility checker

### Description

XLSX reports available for extension, Node and Cypress, but still missing in Karma

**Comment by marcjohlic** [comment]
@ErickRenteria This one was discussed on the standup.  I added the Ready for QA label for you to check - but it may just need to be closed.

**Comment by ErickRenteria** [comment]
Merged
